client/modules/IDE/components/Header/MobileNav.jsx

```diff
@@ -0,0 +1,388 @@
+      transition: transform 100ms 200ms;
+      display: flex;
+      flex-direction: column;
+      gap: ${remSize(2)};
+      box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px,
+        rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
+      min-width: ${remSize(120)};
+      border-radius: ${remSize(2)};
+
+      /* User */
+      li.user {
+        color: ${prop('colors.processingBlue')};
+        font-size: ${remSize(16)};
+        font-weight: bold;
+        margin: ${remSize(6)} ${remSize(12)};
+      }
+
+      > b {
+        margin: ${remSize(6)} ${remSize(12)};
+        position: relative;
+        width: min-content;
+        &::after {
+          content: '';
+          position: absolute;
+          opacity: calc(0.5);
+          top: 50%;
+          transform: translate(100%, -50%);
+          right: ${remSize(-6)};
+          width: ${remSize(55)};
+          height: 1px;
+          background-color: ${prop('Button.primary.default.foreground')};
+        }
+      }
+
+      > li {
+        display: flex;
+        width: 100%;
+        > button {
+          width: 100%;
+          padding: ${remSize(8)} ${remSize(15)} ${remSize(8)} ${remSize(10)};
+          font-size: ${remSize(18)};
+          text-align: left;
+          &:hover {
+            background-color: ${prop('Button.primary.hover.background')};
+            color: ${prop('Button.primary.hover.foreground')};
+          }
+        }
+      }
+    }
+  }
+`;
+
+const MobileNav = (props) => {
+  const project = useSelector((state) => state.project);
+  const user = useSelector((state) => state.user);
+  const [title, setTitle] = useState();
+
+  // use the useNavigate hook in the react router v6
+  const history = useHistory();
+  const navigate = (url) => history.push(url);
+
+  const { pathname } = useLocation();
+  const editorLink = useSelector(selectSketchPath);
+
+  function resolveTitle(path) {
+    switch (path) {
+      case '/':
+        return project.name;
+      case '/login':
+        return 'Login';
+      case '/signup':
+        return 'Signup';
+      case '/account':
+        return 'Account Settings';
+      case '/p5/sketches':
+      case '/p5/collections':
+        return 'Examples';
+      case `/${user.username}/assets`:
+      case `/${user.username}/collections`:
+      case `/${user.username}/sketches`:
+        return 'My Stuff';
+      default:
+        return project.name;
+    }
+  }
+
+  useEffect(() => {
+    setTitle(resolveTitle(pathname));
+  }, [pathname, project]);
+
+  const Logo = AsteriskIcon;
+  return (
+    <Nav>
+      <LogoContainer>
+        <Logo />
+      </LogoContainer>
+      <Title>
+        <h1>{title}</h1>
+        {project?.owner && title === project.name && (
+          <h5>by {project?.owner?.username}</h5>
+        )}
+      </Title>
+      <Options>
+        {user.authenticated ? (
+          <AccoutnMenu />
+        ) : (
+          <div>
+            <IconButton
+              onClick={() => {
```

Comment:
I know we’ve discussed this but these should be links.  👊💀🐴

Reply:
yeah replaced all `navigate()` with `<Link>` 😁